variants/STM32C0xx/C011D6Y_C011F(4-6)(P-U)_C031F(4-6)P/boards_entry.txt

```diff
@@ -4,12 +4,18 @@
 # See: https://github.com/stm32duino/Arduino_Core_STM32/wiki/Add-a-new-variant-%28board%29
 
 # Generic C011D6Yx
+
```

Comment:
This file must not be changed, it is generated automatically. It is used to ease addition in the `boards.txt`.
All generic should be added in the `boards.txt`.